SYCL/ESIMD/vadd_raw_send.cpp

```diff
@@ -7,7 +7,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: gpu
-// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: gpu-intel-dg1,cuda,hip
```

Comment:
Looks Ok/Good, but please add the TODO comment while it is known.
I.e. would be good to copy the description of this PR to the line above, something like:
"TODO: The block read/write function implementation is not compatible on DG1"